Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM golang:1.19 AS tinygo-llvm
 
 RUN apt-get update && \
-    apt-get install -y apt-utils make cmake clang-11 binutils-avr gcc-avr avr-libc ninja-build
+    apt-get install -y apt-utils make cmake clang-11 ninja-build
```

Comment:
Unrelated to this PR, but why is `clang-11` needed?

Reply:
That is a good question. I wonder if it actually is needed now?